rpmlint/configdefaults.toml

```diff
@@ -20,6 +20,7 @@ Checks = [
     "MixedOwnershipCheck",
     "PkgConfigCheck",
     "PostCheck",
+    "SharedLibraryPolicyCheck",
```

Comment:
This one is not for default to be turned on.

Reply:
Yeah, let's not have this on by default. Many RPM distros don't use this policy today.